src/inputStyles/index.css

```diff
@@ -8,12 +8,13 @@ html {
 }
 
 body {
-  @apply antialiased scrolling-touch font-avenirregular m-0 min-h-full relative;
```

Comment:
scrolling-touch is no longer supported after iOS 13, hence Tailwind has removed this class